Fedora/input/system/accounts/pam.xml

```diff
@@ -105,9 +105,9 @@ stringent password strength requirements. It is provided
 in an RPM of the same name and can be configured by setting the configuration
 settings in <tt>/etc/passwdqc.conf</tt>.
 <br /><br />
-The man pages <tt>pam_cracklib(8)</tt> and <tt>pam_passwdqc(8)</tt>
-provide information on the capabilities and configuration of
-each.</description>
+The man pages <tt>pam_pwquality(8)</tt>, <tt>pam_cracklib(8)</tt>, and
+<tt>pam_passwdqc(8)</tt> provide information on the capabilities and
```

Comment:
No, that reference was kept intentionally. While ```pam_passwdqc``` not present in RHEL-7 (not even in the build system), it's (still) present in Fedora:
```
[root@localhost ~]# cat /etc/redhat-release 
Fedora release 20 (Heisenbug)
[root@localhost ~]# rpm -qf /usr/lib64/security/pam_passwdqc.so 
passwdqc-1.3.0-1.fc20.x86_64
```
Corresponding Fedora pkgdb entry is:
&nbsp; &nbsp;  https://admin.fedoraproject.org/pkgdb/package/passwdqc/